### DISCUSSION TITLE
Annotations and labels for alerting rules (template variables) - please add Start/End time of alert

### DISCUSSION THREAD

**Discussion by GedPL** [grafana/grafana#46651] [Alerting]
Hi guys,
Currently we can use following template variables when expanding annotations and labels.
$labels
$values
$value

Is it possible to add StartsAt on alert firing ad EndsAt on alert resolving ? Would be nice to have them in epoch millis.

Thanks a lot !
Ged

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
